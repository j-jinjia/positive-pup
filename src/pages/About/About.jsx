import Layout from "../../components/Layout/Layout";
import AboutUsHeader from "../../components/AboutUsHeader/AboutUsHeader";
import MissionSection from "../../components/MissionSection/MissionSection";
import AboutUsTrainingSection from "../../components/AboutUsTrainingSection/AboutUsTrainingSection";


const About = () => {
  return (
    <Layout>
      <AboutUsHeader />
      <MissionSection />
     <AboutUsTrainingSection />
    </Layout>
  );
};

export default About;
