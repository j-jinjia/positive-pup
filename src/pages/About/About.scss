@use "../../styles/variables" as *;

.certification-section {
  display: flex;
  align-items: center;
  flex-direction: column;
  padding: 2rem;

  &__header {
    font-size: $font-size-h2;
    margin-bottom: 2rem;
    line-height: 1.1;
  }

  &__paragraph {
    margin-bottom: 2rem;
    line-height: 2;
    font-size: 1.25rem;
    color: $colour-tertiary;

    &--highlighted-text {
      color: $colour-primary;
      font-weight: 700;
      text-decoration: underline;
      text-underline-offset: 0.2rem;
    }
  }
}

@media screen and (min-width: $tablet-breakpoint) {
  .certification-section {
    text-align: center;
  }
}

@media screen and (min-width: $desktop-breakpoint) {
  .certification-section {
    max-width: 700px;
  }
}
