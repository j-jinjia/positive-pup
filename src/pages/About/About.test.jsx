import { customRender } from "../../utils/testUtils";
import About from "./About";

it("should render the About page", () => {
  const { container } = customRender(<About />);

  expect(container).toMatchSnapshot();
});
