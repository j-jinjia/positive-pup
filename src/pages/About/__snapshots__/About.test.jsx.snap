// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render the About page 1`] = `
<div>
  <div
    class="layout"
  >
    <div
      class="navbar"
    >
      <div
        class="navbar__container"
      >
        <a
          href="/"
        >
          <img
            alt="The Positive Pup logo"
            class="navbar__container-logo"
            src="footer-logo.svg"
          />
        </a>
        <nav
          class="navbar__nav"
        >
          <a
            class="navbar__nav-item"
            href="/about"
          >
            About
          </a>
          <a
            class="navbar__nav-item"
            href="/services"
          >
            Services
          </a>
          <a
            class="navbar__nav-item"
            href="/testimonials"
          >
            Testimonials
          </a>
          <a
            class="navbar__nav-item"
            href="/gallery"
          >
            Gallery
          </a>
          <span
            class="navbar__nav-item navbar__nav-item--phone"
          >
            <img
              alt=""
              class="navbar__nav-item-button"
              src="phone.svg"
            />
            <a
              class="navbar__nav-item"
              href="/get-in-touch"
            >
              Get in Touch
            </a>
          </span>
          <a
            class="button button--primary"
            href="/book-now"
          >
            Book Now
          </a>
        </nav>
      </div>
    </div>
    <div
      class="about-us-header"
    >
      <main
        class="about-us-header__content"
      >
        <h1
          class="about-us-header__content-title"
        >
          ABOUT US
        </h1>
        <div
          class="hero"
        >
          <h2
            class="hero__header"
          >
            Hey there
            <div>
               
            </div>
            My name is Melanie!
          </h2>
          <p
            class="hero__subheader"
          >
            As the owner of The Positive Pup, it is my joy to be able to welcome you to this website.
          </p>
          <button
            class="button button--secondary"
          >
            More About Us
          </button>
        </div>
      </main>
      <img
        alt="Melanie and a dog"
        class="about-us-header__image"
        src="AboutMe.png"
      />
    </div>
    <div
      class="mission-section"
    >
      <h1
        class="mission-section__heading"
      >
         My Mission
      </h1>
      <p
        class="mission-section__about-us"
      >
        Have you ever dreamt about your perfect career? the kind of job that would make you jump out of bed in the morning, ready to welcome in the new day? Mine has always involved working with dogs and making a difference to people’s lives.
      </p>
      <p
        class="mission-section__about-us"
      >
        I have made it my mission to help promote and strengthen the bond between humans and their canine companions, so that I can make a difference in your world. I truly believe that by increasing the understanding of how dogs communicate with their humans, amazing things can be achieved. The results speak for themselves, content well-mannered canines and relaxed happy humans!
      </p>
      <p
        class="mission-section__about-us"
      >
        Thanks for visiting The Positive Pup and for taking your very first step towards building a special bond with your canine companion.
      </p>
      <div
        class="link-container"
      >
        <a
          class="arrow-link"
          href="/"
        >
          View Training Methods
          <img
            alt=""
            class="arrow-link__arrow"
            src="arrow.svg"
          />
        </a>
      </div>
    </div>
    <div
      class="about-us-training-section"
    >
      <img
        alt="Positive Pup owner, Melanie, with a dog"
        class="about-us-training-section__image"
        src="trainingMethodsImage.svg"
      />
      <div
        class="about-us-training-section__content"
      >
        <h2
          class="about-us-training-section__header"
        >
          Training Methods 
        </h2>
        <p
          class="about-us-training-section__paragraph"
        >
          As a graduate of the Victoria Stilwell Dog Training Academy, I am a positive reinforcement dog trainer. This means that I use reward-based training methods that are deeply rooted in the continually evolving world of behavioral science.
        </p>
        <p
          class="about-us-training-section__paragraph"
        >
          Outdated old school dominance and punishment are certainly not my bag. Not only does science tell us that these methods achieve temporary results and can often result in eventual aggressive behavior, but that they are cruel, dangerous and unnecessary.
        </p>
      </div>
    </div>
    <div
      class="certifications"
    >
      <h2
        class="certifications__header"
      >
        You’re in professional hands!
      </h2>
      <p
        class="certifications__description"
      >
        I’m a well-established, certified trainer in Dog behaviour & training, with over
         
        <span
          class="certifications__description--highlighted-text"
        >
          4.5 years of experience
        </span>
        . To find out more information about any of my certifications, please click on one of the logos.
      </p>
      <div
        class="certifications__badge-container"
      >
        <a
          href="https://www.vsdogtrainingacademy.com/"
          rel="noreferrer"
          target="_blank"
        >
          <img
            alt="Victoria Stillwell certification badge"
            class="certifications__badge"
            src="victoria_stillwell.png"
          />
        </a>
        <a
          href="https://companionanimal.network/"
          rel="noreferrer"
          target="_blank"
        >
          <img
            alt="international companion animal network certification badge"
            class="certifications__badge"
            src="international_companion.png"
          />
        </a>
        <a
          href="https://ukdogcharter.org/"
          rel="noreferrer"
          target="_blank"
        >
          <img
            alt="UK dog behaviour and training charter certification badge"
            class="certifications__badge"
            src="UK_dog_behaviour.png"
          />
        </a>
        <a
          href="https://www.petprofessionalguild.com/"
          rel="noreferrer"
          target="_blank"
        >
          <img
            alt="the pet professional guild certification badge"
            class="certifications__badge"
            src="PPG_Member_Badge.png"
          />
        </a>
      </div>
    </div>
    <div
      class="footer"
    >
      <div
        class="footer__image-container"
      >
        <img
          alt="logo"
          class="footer__logo"
          src="footer-logo.svg"
        />
      </div>
      <ul
        class="footer__links-container"
      >
        <li
          class="footer__list-item"
        >
          <a
            class="footer__link"
            href="/about"
          >
            About
          </a>
        </li>
        <li
          class="footer__list-item"
        >
          <a
            class="footer__link"
            href="/services"
          >
            Services
          </a>
        </li>
        <li
          class="footer__list-item"
        >
          <a
            class="footer__link"
            href="/gallery"
          >
            Gallery
          </a>
        </li>
        <li
          class="footer__list-item footer__list-item--desktop"
        >
          <a
            class="footer__link"
            href="/terms-and-conditions"
          >
            Terms & Conditions
          </a>
        </li>
        <li
          class="footer__list-item footer__list-item--mobile"
        >
          <a
            class="footer__link"
            href="/terms-and-conditions"
          >
            T&C's
          </a>
        </li>
        <li
          class="footer__list-item"
        >
          <a
            class="footer__link"
            href="/faqs"
          >
            FAQ's
          </a>
        </li>
        <li
          class="footer__list-item"
        >
          <a
            class="footer__link"
            href="/get-in-touch"
          >
            Contact
          </a>
        </li>
      </ul>
      <hr
        class="footer__line-break"
      />
      <div
        class="footer__secondary-links-container"
      >
        <div
          class="footer__copyright-text"
        >
          © 2022 The Positive Pup. All rights reserved.
        </div>
        <div
          class="footer__social-links"
        >
          <a
            href="https://www.facebook.com/thepositivepupdogtraining"
            target="blank"
          >
            <img
              alt="link to facebook"
              src="facebook-logo.svg"
            />
          </a>
          <a
            href="https://www.instagram.com/the_positivepup/"
            target="blank"
          >
            <img
              alt="link to instagram"
              src="instagram-logo.svg"
            />
          </a>
        </div>
      </div>
    </div>
  </div>
</div>
`;
